Reporter.py

```diff
@@ -56,10 +56,8 @@ def report_runtime(rightsize: bool, runtime_results: RuntimeReport) -> None:
     def print_prelude():
         print(colored(banner, 'yellow'))
         print(f"""
-For continuous scanning of your environment to detect drifts, connect to the {colored("FREE", 'green', attrs=['bold'])} {colored("Bridgecrew", 'magenta', attrs=['bold'])} platform
-Check us out!
-https://www.bridgecrew.cloud
-https://www.bridgecrew.io
+For continuous scanning of your environment to detect drifts, connect to the {colored("Bridgecrew", 'magenta', attrs=['bold'])} platform
+Check us out - https://www.bridgecrew.cloud
```

Comment:
take a look at : https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda

Reply:
@schosterbarak Can't get this to work... Not in python and not on my terminal